### PR TITLE
[WIP] complete LF_oat + tmle3_spec_oat + test

### PR DESCRIPTION
**Pull request by wilsoncai1992**

### CODE REVIEW

File: R/LF_oat.R [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      set(Q_values, , "A",A_values)
+      g_task <- sl3::make_sl3_Task(
+        Q_values,
+        outcome = "A",
+        covariates = covariates
+        )
+      return(g_task)
+    }
+  ),
+  # WILSON: i am not sure to keep the followings or not?
```
Comment (wilsoncai1992) [inline R/LF_oat.R:136]: @jeremyrcoyle any suggestions?

File: R/LF_oat.R [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      learner_task <- self$create_regression_task(
+        self$LF_Q,
+        self$tmle_task_training,
+        self$tmle_task_training
+        )
+      learner_fit <- delayed_learner_train(self$learner, learner_task)
+      return(learner_fit)
+    },
+    get_mean = function(tmle_task, cv_fold) {
+      # WILSON: I am not sure if user should only call this method directly,
```
Comment (wilsoncai1992) [inline R/LF_oat.R:73]: @jeremyrcoyle welcome suggestions on API styling

**Comment by wilsoncai1992** [comment]
in terms of next step, should i write a new tmle_spec?

**Comment by wilsoncai1992** [comment]
The whole pipeline is passing. 
documentation remains to be completed.
more realistic simulation (+ test) will be addressed in separate PR.

**Comment by wilsoncai1992** [comment]
@jeremyrcoyle some weird installation bug is happening in travis and appveyor. Can you take a look and let me know how to fix?

**Comment by wilsoncai1992** [comment]
package 'backports' successfully unpacked and MD5 sums checked
Error: (converted from warning) cannot remove prior installation of package 'backports'
Execution halted
Command exited with code 1
7z a failure.zip *.Rcheck\*
7-Zip 18.05 (x64) : Copyright (c) 1999-2018 Igor Pavlov : 2018-04-30
Scanning the drive:
0 files, 0 bytes
Creating archive: failure.zip
Add new data to archive: 0 files, 0 bytes

**Comment by jeremyrcoyle** [comment]
The log above is an issue with devtools. I fixed it in tmle3 with this commit: https://github.com/tlverse/tmle3/commit/6245a5786369909045326fdf5040ba9011db9944. Appveyor is broken currently for all my repos and I can't figure out why. I've just been ignoring it and hoping it gets resolved.

**Comment by wilsoncai1992** [comment]
Thanks Jeremy! Curious to know when this will be fixed

**Comment by jeremyrcoyle** [comment]
https://github.com/tlverse/tmle3/pull/29 looks like my latest just passed, so maybe it's resolved itself?

**Pull request merged**